app-tasks/dags/geotiff/find_geotiff_scenes.py

```diff
@@ -0,0 +1,102 @@
+from collections import namedtuple
+import json
+import logging
+import os
+from dateutil import parser
+import boto3
+from datetime import datetime
+
+from airflow.models import DAG
+from airflow.bin.cli import trigger_dag
+from airflow.operators.python_operator import PythonOperator
+
+from rf.uploads.geotiff import GeoTiffS3SceneFactory
+from rf.utils.io import Visibility
+
+rf_logger = logging.getLogger('rf')
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+rf_logger.addHandler(ch)
+
+logger = logging.getLogger(__name__)
+
+args = {
+    'start_date': datetime(2017, 2, 21)
```

Comment:
Not really sure what I should be putting here, but it complains if I don't have it.